snippet-extractor-output/snippets.yaml

```
@@ -10134,6 +10134,77 @@
         console.log("masterCategories.addAsync call failed with error: " + asyncResult.error.message);
       }
     });
+'Office.MailboxEnums.ComposeType:enum':
+  - >-
+    // Link to full sample:
+    https://raw.githubusercontent.com/OfficeDev/office-js-snippets/prod/samples/outlook/60-event-based-activation/work-with-client-signatures.yaml
+
+    // Get the compose type of the current item (mail only).
```

if you accept this, you'll need to update in the source yaml
```suggestion
    // Get the compose type of the current message.
```